server/index.js

```diff
@@ -1,11 +1,15 @@
+/* load configs */
+require('dotenv').config()
+require('./config/passport-setup')
+
+/* app setup */
 const app = require('./app')
-const dotenv = require('dotenv')
-dotenv.config()
+const db = require('./db')
 
 const PORT = process.env.PORT || 8000
 
 try {
-    app.listen(PORT, () => console.log(`\nBackend service running on port ${PORT}\n`))
+    db.sync().then(app.listen(PORT, () => console.log(`\nBackend service running on port ${PORT}\n`)))
```

Comment:
because we're using the .then syntax here, how do you feel about just using a .catch instead of wrapping the whole thing in a try-catch block?